### ISSUE TITLE
Problem with conversion of torch model to onnx

### ISSUE THREAD

**Issue by gianlourbano** [pytorch/pytorch#135343]
### 🐛 Describe the bug

I'm trying to convert [HTDemucs](https://github.com/adefossez/demucs) to onnx with dynamo export. The plain model gives me a bunch of errors, complaining about the istft operator, so i placed both the latter and the stft outside of it (model now expects both the audio mix and its spectrogram as inputs). It then succesfully converts, but apparently there are some problems with the accuracy of the newly converted model.

```
W0906 11:44:51.977000 18782 torch/onnx/_internal/exporter/_core.py:1290] Output 'add_76' has a large absolute difference of 2.171368. 
W0906 11:44:51.978000 18782 torch/onnx/_internal/exporter/_core.py:1297] Output 'add_76' has a large relative difference of 875822.000000. 
W0906 11:44:51.978000 18782 torch/onnx/_internal/exporter/_core.py:1290] Output 'add_77' has a large absolute difference of 0.011793. 
W0906 11:44:51.978000 18782 torch/onnx/_internal/exporter/_core.py:1297] Output 'add_77' has a large relative difference of 4314.738770. 
```

[Here](https://github.com/gianlourbano/demucs) is the modified version of the model with the [conversion script](https://github.com/gianlourbano/demucs/blob/main/convert_to_onnx.py) used.

### Versions

PyTorch version: 2.5.0.dev20240828+cpu
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: Ubuntu 24.04.1 LTS (x86_64)
GCC version: (Ubuntu 13.2.0-23ubuntu4) 13.2.0
Clang version: 18.1.8 (++20240731025043+3b5b5c1ec4a3-1~exp1~20240731145144.92)
CMake version: version 3.30.1
Libc version: glibc-2.39

Python version: 3.10.14 (main, May  6 2024, 19:42:50) [GCC 11.2.0] (64-bit runtime)
Python platform: Linux-6.8.0-41-generic-x86_64-with-glibc2.39
Is CUDA available: False
CUDA runtime version: No CUDA
CUDA_MODULE_LOADING set to: N/A
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

CPU:
Architecture:                         x86_64
CPU op-mode(s):                       32-bit, 64-bit
Address sizes:                        48 bits physical, 48 bits virtual
Byte Order:                           Little Endian
CPU(s):                               12
On-line CPU(s) list:                  0-11
Vendor ID:                            AuthenticAMD
Model name:                           AMD Ryzen 5 4600H with Radeon Graphics
CPU family:                           23
Model:                                96
Thread(s) per core:                   2
Core(s) per socket:                   6
Socket(s):                            1
Stepping:                             1
Frequency boost:                      enabled
CPU(s) scaling MHz:                   74%
CPU max MHz:                          3000,0000
CPU min MHz:                          1400,0000
BogoMIPS:                             5988,52
Flags:                                fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx mmxext fxsr_opt pdpe1gb rdtscp lm constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid aperfmperf rapl pni pclmulqdq monitor ssse3 fma cx16 sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt tce topoext perfctr_core perfctr_nb bpext perfctr_llc mwaitx cpb cat_l3 cdp_l3 hw_pstate ssbd mba ibrs ibpb stibp vmmcall fsgsbase bmi1 avx2 smep bmi2 cqm rdt_a rdseed adx smap clflushopt clwb sha_ni xsaveopt xsavec xgetbv1 cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local clzero irperf xsaveerptr rdpru wbnoinvd cppc arat npt lbrv svm_lock nrip_save tsc_scale vmcb_clean flushbyasid decodeassists pausefilter pfthreshold avic v_vmsave_vmload vgif v_spec_ctrl umip rdpid overflow_recov succor smca
Virtualization:                       AMD-V
L1d cache:                            192 KiB (6 instances)
L1i cache:                            192 KiB (6 instances)
L2 cache:                             3 MiB (6 instances)
L3 cache:                             8 MiB (2 instances)
NUMA node(s):                         1
NUMA node0 CPU(s):                    0-11
Vulnerability Gather data sampling:   Not affected
Vulnerability Itlb multihit:          Not affected
Vulnerability L1tf:                   Not affected
Vulnerability Mds:                    Not affected
Vulnerability Meltdown:               Not affected
Vulnerability Mmio stale data:        Not affected
Vulnerability Reg file data sampling: Not affected
Vulnerability Retbleed:               Mitigation; untrained return thunk; SMT enabled with STIBP protection
Vulnerability Spec rstack overflow:   Mitigation; Safe RET
Vulnerability Spec store bypass:      Mitigation; Speculative Store Bypass disabled via prctl
Vulnerability Spectre v1:             Mitigation; usercopy/swapgs barriers and __user pointer sanitization
Vulnerability Spectre v2:             Mitigation; Retpolines; IBPB conditional; STIBP always-on; RSB filling; PBRSB-eIBRS Not affected; BHI Not affected
Vulnerability Srbds:                  Not affected
Vulnerability Tsx async abort:        Not affected

Versions of relevant libraries:
[pip3] numpy==1.26.0
[pip3] onnx==1.16.2
[pip3] onnxruntime==1.19.0
[pip3] onnxscript==0.1.0.dev20240902
[pip3] torch==2.5.0.dev20240828+cpu
[pip3] torchaudio==2.5.0.dev20240828+cpu
[pip3] torchvision==0.20.0.dev20240828+cpu
[conda] torch                     2.5.0.dev20240828+cpu          pypi_0    pypi
[conda] torchaudio                2.5.0.dev20240828+cpu          pypi_0    pypi
[conda] torchvision               0.20.0.dev20240828+cpu          pypi_0    pypi

**Comment by justinchuby** [comment]
Could you share the generated report? Thanks!

**Comment by gianlourbano** [comment]
Of course! I have updated the repo, [here it is] (https://github.com/gianlourbano/demucs/blob/main/onnx_export_2024-09-08_18-33-35-664327_accuracy.md). Let me know if you're interested also in the report of the original model

**Comment by gianlourbano** [comment]
@justinchuby further testing with the converted model i would say there are no problems with the accuracy. However, i'm facing a new problem with the conversion of a modified version of this model, that i previously did not have. The std torch operator was previously decomposed with aten (in version torch==2.5.0.dev20240828+cpu, onnxscript==0.1.0.dev20240902) while in version torch==2.6.0.dev20241221, onnxscript==0.1.0.dev2024122 it's decomposed with prims, giving this error:

```
Traceback (most recent call last):

  File "/Users/gianlorenz.ourbanomusixmatch.com/miniconda3/envs/demucs-onnx/lib/python3.10/site-packages/torch/onnx/_internal/exporter/_core.py", line 708, in _translate_fx_graph
    _handle_call_function_node_with_lowering(

  File "/Users/gianlorenz.ourbanomusixmatch.com/miniconda3/envs/demucs-onnx/lib/python3.10/site-packages/torch/onnx/_internal/exporter/_core.py", line 490, in _handle_call_function_node_with_lowering
    raise _errors.DispatchError(

torch.onnx._internal.exporter._errors.DispatchError: No ONNX function found for <OpOverload(op='prims.var', overload='default')>. Failure message: No decompositions registered for the real-valued input
```
